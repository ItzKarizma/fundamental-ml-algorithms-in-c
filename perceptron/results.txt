23.000000 42.500000 0
30.000000 58.100000 1
45.000000 52.000000 0
58.000000 89.200000 1
29.000000 39.700000 0
50.000000 78.500000 1
33.000000 48.900000 0
65.000000 95.000000 1
20.000000 35.100000 0
42.000000 68.300000 1
