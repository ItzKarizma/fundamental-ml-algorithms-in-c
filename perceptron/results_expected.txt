23 42.5 0
30 58.1 1
45 52.0 0
58 89.2 1
29 39.7 0
50 78.5 1
33 48.9 0
65 95.0 1
20 35.1 0
42 68.3 1
